tests/formatter.rs

```diff
@@ -2937,7 +2937,7 @@ error: title
     assert_data_eq!(renderer.render(input), expected_ascii);
 
     let expected_unicode = str![[r#"
-error: title
+error: titlecompiler/rustc_errors
```

Comment:
does this commit pass on its own?  I'm not seeing the edit that would cause this output change (and ascii is unchanged)

Reply:
That change shouldn't have been there. I'm not sure how I missed it.